javers-core/src/main/java/org/javers/common/reflection/JaversFieldFactory.java

```diff
@@ -34,7 +41,20 @@ public JaversFieldFactory(Class methodSource) {
         return fields;
     }
 
-    private JaversField createJField(Field rawField, TypeResolvingContext context){
+    public JaversField getField(String fieldName) {
```

Comment:
tak, w moim przypadku nie potrzebuje tej logiki dziedziczenia, wykorzystuje to tylko do klas SetWrapper i ListWrapper